### ISSUE TITLE
license

### ISSUE THREAD

**Issue by hansinator** [das-labor/nixie-clock#2]
Hi,

I need to know how this is licensed.

**Comment by e7p** [comment]
This project is licensed with GPL3. Look into the LICENSE file in the tree (or das-labor wiki) for more details.
As this was only made from rework based on research (and some reverse engineering of the original project), I felt free to choose this license.

**Comment by hansinator** [comment]
I seem to have posted this issue in the wrong browser tab.. My original intention was to ask about the micro lib that Zaolin uploaded. However, I am asking specifically about the CAN code. It seems to me that you are neither the original author nor have rewritten it from scratch.

By the way, here is the original project page from the original maker of the nixie-clock you are working on.. including a schematic:
http://wiki.carluccio.de/index.php/Nixie_Clock

**Comment by e7p** [comment]
Okay. The included can lib is for sure the one from microlib, so not included by the license. It could be made more clear by adding it as submodule, this would be good I suppose, but I don't really know how to...

Ah, I did not know, that dario has a wikipage on his site, but in the meantime I also found code in the labor svn. I just had it complete at this time, so I thought it would not be so bad to upload my more clean code. I also asked the original authors of the project, and they granted me the permission to publish it under the GPL3.
